integration/PhpSpec/Console/Prompter/QuestionTest.php

```diff
@@ -3,12 +3,13 @@
 namespace integration\PhpSpec\Console\Prompter;
 
 use PhpSpec\Console\Prompter\Question;
+use PHPUnit\Framework\TestCase;
```

Comment:
Thanks for that.  I've upgraded the minimum PHPUnit version to `5.5`, so the build passes with `composer update --prefer lowest`.